Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-network-operator
 COPY . .
-RUN hack/build-go.sh
+RUN hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/cluster-network-operator/_output/linux/$(go env GOARCH)/cluster-network-operator /tmp/build/; \
+    cp /go/src/github.com/openshift/cluster-network-operator/_output/linux/$(go env GOARCH)/cluster-network-renderer /tmp/build/
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
-COPY --from=builder /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-operator /usr/bin/
```

Comment:
Why not just use $(go env GOARCH) here? Does /tmp/build add anything?

Reply:
@dcbw Yeah, but that would require `golang` to be present in the base i.e. registry.svc.ci.openshift.org/openshift/origin-v4.0:base image.